runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/woodcutting/CrowdsourcingWoodcutting.java

```diff
@@ -55,57 +55,123 @@
 	private static final String INVENTORY_FULL_MESSAGE = "Your inventory is too full to hold any more logs.";
 	private static final String NEST_MESSAGE = "A bird's nest falls out of the tree";
 	private static final Set<Integer> TREE_OBJECTS = new ImmutableSet.Builder<Integer>().
-		add(ObjectID.OAK_10820).
-		add(ObjectID.YEW).
-		add(ObjectID.TREE).
-		add(ObjectID.TREE_1278).
-		add(ObjectID.WILLOW).
-		add(ObjectID.WILLOW_10829).
-		add(ObjectID.WILLOW_10831).
-		add(ObjectID.WILLOW_10833).
-		add(ObjectID.SCRAPEY_TREE).
-		add(ObjectID.JUNGLE_TREE_15951).
-		add(ObjectID.JUNGLE_TREE_15954).
-		add(ObjectID.JUNGLE_TREE_15948).
-		add(ObjectID.MAPLE_TREE_10832).
-		add(ObjectID.MAHOGANY).
-		add(ObjectID.TEAK).
-		add(ObjectID.MAGIC_TREE_10834).
-		add(ObjectID.HOLLOW_TREE_10821).
-		add(ObjectID.HOLLOW_TREE_10830).
-		add(ObjectID.ACHEY_TREE).
-		build();
+			add(ObjectID.OAK).
```

Comment:
Indentation needs to be fixed in this file. All these changed lines should have two leading tabs, not three.